crates/wasm-encoder/src/exports.rs

```diff
@@ -80,10 +80,14 @@ pub enum Export {
     Memory(u32),
     /// An export of the `n`th global.
     Global(u32),
+    /// An export of the `n`th instance.
+    Instance(u32),
+    /// An export of the `n`th module.
+    Module(u32),
```

Comment:
Can you mention in the doc comments for these that they are part of the module linking proposal?